apps/aecore/src/aec_conductor.erl

```diff
@@ -86,6 +86,7 @@
 -define(SERVER, ?MODULE).
 
 -define(DEFAULT_MINING_ATTEMPT_TIMEOUT, 60 * 60 * 1000). %% milliseconds
+-define(NONCE_RANGE, 1000000000000000000000000). %% 10^24, approx. 2^80
```

Comment:
Why use a constant that is bigger than the max allowed nonce?

Reply:
You are correct. I corrected force pushing new commit. Thanks.